config.template.py

```diff
@@ -40,6 +40,12 @@ class Config:
     # "tashiSSH", and "ec2SSH"
     VMMS_NAME = "localDocker"
 
+    # Update this to the 'volumes' directory of your Tango installation if
+    # Docker is being used as the VMMs.
+    # It must be an absolute path with trailing slash, i.e
+    # /opt/TangoService/Tango/volumes/
+    DOCKER_VOLUME_PATH = ""
```

Comment:
I understand this has been merged into master already, but why deleting the default volume path instead of appending the trailing slash to the previous string `/opt/TangoService/Tango/volumes`? This actually complicates the deployment a little bit.

Reply:
The reason is that for non-Dockerfile setups this may be confusing for new users as it may seem to suggest that the Tango installation must be located in /opt/TangoService

Reply:
The reason is that for non-Dockerfile setups this may be confusing for new users as it may seem to suggest that the Tango installation must be located in /opt/TangoService